### PR TITLE
Cleaning up rpc method names

### PR DESCRIPTION
**Pull request by Elfoniok**
Signed-off-by: Lukasz Foniok <lukaszfoniok@gmail.com>

### CODE REVIEW

**Review by Krigpl — APPROVED** [review]

**Comment by etam** [comment]
It's a good practice to rename files in one commit and change them in second. This way git better tracks renames. (In this case I need to use "git diff -M10%" to see what changed in renamed files)

File: golem/rpc/mapping/rpcmethodnames.py [hunk 1]
```diff
@@ -0,0 +1,88 @@
+CORE_METHOD_MAP = dict(
+    get_golem_version=      'golem.version',
```
Comment (etam) [inline golem/rpc/mapping/rpcmethodnames.py:2]: Why not aliases, like it was before?

This way it's easier to make mistake. You'll spend hours debugging if there's mismatch in strings. You'll get instant error if object does not exist.

Comment (etam) [reply golem/rpc/mapping/rpcmethodnames.py:2]: Yep. You removed 'golem.version' from the other file.

**Review by etam — CHANGES_REQUESTED** [review]

**Review by jiivan — APPROVED** [review]
Great! Maybe in a next step change meaningless classes in `golem/rpc/mapping/rpceventnames.py` to plain dicts?

**Comment by etam** [comment]
If we have constant set of methods, then using dict is not a good idea. Better use enums, or leave it as is.

File: golem/rpc/mapping/rpcmethodnames.py (hunk 1 above)
Comment (Elfoniok) [reply golem/rpc/mapping/rpcmethodnames.py:2]: You will not, there is not point for using those in code, that is why events, are left as aliases, but method names are passed to crossbar ruter only. You don't need them unless you are using debug class, but i guess then you know what are you doing and can use those correctly.

**Review by etam — APPROVED** [review]

**Pull request merged**